src/QMCWaveFunctions/TWFdispatcher.h

```diff
@@ -29,6 +28,7 @@ class TWFdispatcher
   using ComputeType  = TrialWaveFunction::ComputeType;
   using ValueType    = TrialWaveFunction::ValueType;
   using GradType     = TrialWaveFunction::GradType;
+  using ComplexType  = TrialWaveFunction::ComplexType;
```

Comment:
This is new code and should follow current type naming conventions, `Type` is redundant.   Since `TWF::ComplexType` is just the complex normal precision value I would just do `using Complex = TrialWaveFunction::ComplexType`

You could argue its better if its `QMCTraits::ComplexType` is the source for the alias but that's a bigger mess.

Reply:
done. 

Note that I only did it for TWFdispatcher. I didn't change the ComplexType definitions in TrialWaveFunction or WaveFunctionComponents which would require changes all over the place